engine/classes/Elgg/Notifications/InstantNotificationEventHandler.php

```diff
@@ -32,9 +63,44 @@ public function getSubscriptions(): array {
 	}
 	
 	/**
-	 * {@inheritDoc}
+	 * {@inheritdoc}
+	 */
+	protected function getNotificationSubject(\ElggUser $recipient, string $method): string {
```

Comment:
need to rethink these (as they are not allowing to overrule from params) (similar issue as methods override)